IgdbApiWrapper.js

```diff
@@ -13,9 +15,62 @@ function IgdbWrapper(){
 };
 
 IgdbWrapper.prototype.getGamesData = async function(){
-    let response = await request.get(this.options);
+    const gamesEndpointOptions = this.makeEndpointOptions({
+        path: 'games/',
+        queryParams: {
+            fields: ['name', 'popularity', 'cover.url', 'id'],
+            order: 'popularity:desc'    
+        }
+      }
+    );
+    let response = await request.get(gamesEndpointOptions);
     let gamesData = JSON.parse(response);
     return gamesData;    
 };
+IgdbWrapper.prototype.getGameData = async function(gameId){
+    const gameEndpointOptions = this.makeEndpointOptions({
+        path: `${"games/" + gameId}`,
+        queryParams: {
+            fields: ['name', 'summary', 'first_release_date', 'platforms',
+                'developers', 'websites', 'popularity', 'cover.cloudinary_id']
+        }
+    });
+    let response = await request.get(gameEndpointOptions);
+    let gameData = JSON.parse(response);
+    return gameData;
+};
+IgdbWrapper.prototype.makeEndpointOptions = function(endpointParams){
+    let endpointOptions = Object.assign({}, this.options);
+    endpointOptions.url = buildUrl(this.options.url, endpointParams);
```

Comment:
This line could be optimized and moved to the Object.assign() function. Refer to Object.assign() documentation and do the optimization.